assets/src/story-embed-block/block/storyPlayer.js

```diff
@@ -24,13 +24,15 @@ import PropTypes from 'prop-types';
  */
 import { forwardRef } from '@wordpress/element';
 
-function StoryPlayer({ url, title, poster, width, height }, ref) {
+function StoryPlayer({ url, title, poster, width, height, align }, ref) {
+  const margin = 'center' === align ? 'auto' : '0';
   return (
     <amp-story-player
       ref={ref}
       style={{
         width: width ? `${width}px` : undefined,
         height: height ? `${height}px` : undefined,
+        margin: margin,
       }}
       data-testid="amp-story-player"
```

Comment:
Note to self: check whether it makes sense to remove this

Reply:
This keeps the markup the same as generated in PHP.

Reply:
Wrong thread? This is about `data-testid`, and I assume you were replying to my question about `margin`